net-libs/tox/tox-9999.ebuild

```diff
@@ -2,18 +2,18 @@
 # Distributed under the terms of the GNU General Public License v2
 # $Id$
 
-EAPI=5
+EAPI=6
 
 inherit autotools eutils git-r3 user systemd
```

Comment:
Do you need the magic in `prune_libtool_files`, or can you just unconditionally throw away all of them? If the latter, you could replace `prune_libtool_files` with `find -name '*.la' -delete || die` and I think you'd no longer use eutils then.